prereq_tools/base.py

```diff
@@ -64,6 +64,151 @@
 else:
     import configparser as ConfigParser
 
+class EnvironmentModule():
+    """Class for utilizing Modules component to load environment modules"""
+
+    def __init__(self):
+        """Load Modules for initializing envirables"""
+        self._module_func = self._init_module_func(lambda *x: False)
+        self._module_load = self._init_mpi_module()
+
+    def set_module_func(self, func):
```

Comment:
where is this function used?

Reply:
Hmm, maybe not...it might be a holdover...I'll update the patch